models/reduce_pretrained_ft.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        words = word_tokenize(preprocess(row.title + " SEP " + row.body if type(row.body) == str else row.title))
</pre><p>Could you please add an explanation here? Like a comment that explains why are we doing this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/github-labeler/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>